CHANGELOG.md

```diff
@@ -2,7 +2,9 @@
 
 ## Current (in progress)
 
-- Nothing yet
+- :warning: Deletion workflow changes [#2488](https://github.com/opendatateam/udata/pull/2488):
+  - Deleting a resource now triggers the deletion of the corresponding static file
+  - Deleting a dataset now triggers the deletion of the corresponding community resources and their static files
```

Comment:
add org and user too, plus a reference to the migration script and its impacts

(good CHANGELOG entry 👏)